src/Umbraco.Web.UI.Client/src/common/directives/components/tags/umbtagseditor.directive.js

```diff
@@ -270,8 +271,12 @@
                 return ($.inArray(suggestion.value, vm.viewModel) === -1);
             });
         }
-
-
+        
+        function reValidate() {
+            //this is required to re-validate
+            vm.tagEditorForm.tagCount.$setViewValue(vm.viewModel.length);
+        {
```

Comment:
I had to fix this to `}`

Reply:
also there still was a bug, in `validateMandatory` to so have added `|| (vm.value != null && vm.value.length > 0)`
This is nessasary when you use the vm.viewModel.length as model

Reply:
Thanks for the fix, note to self, don't code directly on GitHub again.